docs/api-auth.md

```diff
@@ -25,12 +27,94 @@ There are three different ways a user is considered an admin:
 
 Users (by default) are given access to the leasing and usage APIs.  This is done so they can request their own lease and look at the usage of their leases.  Any appropriately authenticated user in Cognito will automatically fall into the `Users` role.
 
+### Configuring Cognito for Usage with the DCE CLI
```

Comment:
Scrap _with the DCE CLI_ from this section heading.

We use the DCE CLI to demonstrate how DCE works, but this section is really about _Configuring DCE Users in Cognito_